content/usgs-packages/geoknife_Execute.Rmd

```diff
@@ -0,0 +1,147 @@
+
+```{r create-geojob, message=FALSE, warning=FALSE}
+# create fabric
+evap_fabric_info <- list(times = as.POSIXct(c("2005-01-01", "2015-01-01")),
+                         variables = "et",
+                         url = 'https://cida.usgs.gov/thredds/dodsC/ssebopeta/yearly')
+evap_fabric <- webdata(evap_fabric_info)
+
+# create stencil
+evap_stencil <- webgeom('state::Indiana')
+
+# create knife (which defaults to weighted)
+evap_knife <- webprocess()
+# find unweighted algorithm
+all_algorithms <- query(evap_knife, 'algorithms')
+unw_algorithm <- all_algorithms[grep('unweighted', names(all_algorithms))]
+# set knife algorithm to unweighted
+algorithm(evap_knife) <- unw_algorithm
+
+# create the geojob
+evap_geojob <- geoknife(evap_stencil, evap_fabric, evap_knife)
+```
+
+```{r wait-until-finished, echo=FALSE}
+# need this in order to knit without an error when using result()
+wait(evap_geojob, sleep.time = 10)
+```
+
+## Checking the geojob status
+
+The geojob has been created and started on the server. Now, you can check to see if the processing is complete by using `check`.
+
+```{r check-geojob, message=FALSE, warning=FALSE}
+check(evap_geojob)
+```
+
+Other helpful functions to get status information about the job are `error` (returns T/F to say if there was an error during the processing) and `successful` (returns T/F indicating whether the job process was able to complete without any issues). Only one of these can return `TRUE` at a time.
+
+```{r other-checks-geojob, message=FALSE, warning=FALSE}
+error(evap_geojob)
+successful(evap_geojob)
+```
+
+The results of all the status checks say that our job was successful!
+
+## Getting geojob data
+
+Since this job has finished processing and was successful, you can now get the data. You'll notice that `evap_geojob` does not actually contain any data. It only contains information about the job that you submitted. To get the data, you need to use `result` or `download`. The stock statistics algorithms will return simple tabular data, so you can use `result` to automatically take the output and parse it into an R `data.frame`. We used a basic stat algorithm in the evapotranspiration example, so let's use `result` to get the `geojob` output.
+
+```{r get-geojob, message=FALSE, warning=FALSE}
+evap_data <- result(evap_geojob)
+nrow(evap_data)
+head(evap_data)
+```
+
+Other stock algorithms could return netcdf or geotiff data. This will require you to handle the output manually using `download`. This will allow you to download the output to a file and then read it using your preferred method (e.g. `read.table`, `read.csv`). See `?download` for more information.
```

Comment:
How about: 
```
Other feature summary algorithms could return netcdf or geotiff data. This will require you to handle the output manually using `download`. Use this function to download the output to a file and then read it using your preferred data parsing method. `download` can also be used for tabular data if you have a parsing method that differs from what is used in `result`. See `?download` for more information.
```